locationsyncservice/src/main/java/com/shuttl/location_pings/service/MockLocationSaveService.kt

```diff
@@ -51,21 +58,24 @@ class MockLocationSaveService : Service() {
     override fun onCreate() {
         startForeground(1, notification(this, "Updating mock location details..."))
         Log.d(TAG, "onCreate : Start foreground service to sync mock location")
-        work()
     }
 
     override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
+        var bundle :Bundle ?= intent?.extras
+        configs = bundle!!.getParcelable("config")!!
+        work()
```

Comment:
Calling work() inside onStartCommand() to get custom LocationConfig(). Otherwise getting default value of LocationConfig(minTimeInterval, minDistanceInterval )